lib/administrate/field/deferred.rb

```diff
@@ -11,7 +11,9 @@ def initialize(deferred_class, options = {})
       attr_reader :deferred_class, :options
 
       def new(*args)
-        deferred_class.new(*args, options)
+        # If the args already end with a "keyword hash", merge it with options
```

Comment:
I'm happy to do that - just wondering why you'd like it to go? To me what's happening on the following line is not intuitive at all, hence the comment.